README.md

```diff
@@ -94,6 +94,7 @@ imported modules and code clones. It uses [PyLint](https://www.pylint.org/) and
 passwords and weak cryptographic key size. It relies on [Bandit](https://github.com/PyCQA/bandit).
 - **CoLic** scans the code to extract license information. It currently supports [Nomos](https://github.com/fossology/fossology/tree/master/src/nomos) and [ScanCode](https://github.com/nexB/scancode-toolkit). They can be
 activated by passing the corresponding category: `code_license_nomos` or `code_license_scancode`.
+- **CoLang** gathers insights about code language distribution of a git repository. It relies on [Linguist](https://github.com/github/linguist).
```

Comment:
Could you complete the doc by adding the tools and categories (as similarly done for `CoLic`) ?

Reply:
The [requirements](https://github.com/chaoss/grimoirelab-graal#requirements) section in the readme should be also enhanced to include linguist info. What do you think ?

Reply:
> Could you complete the doc by adding the tools and categories (as similarly done for CoLic) ?

Sure. I'll update it.

> The requirements section in the readme should be also enhanced to include linguist info. What do you think ?

Yes. Good idea. I'll add those :)